content/en/tests/containers.md

```diff
@@ -19,283 +19,387 @@ If you run your tests inside a container that you launch yourself within the bui
+| `CF_BUILD_ID`                | The unique ID of the build.                                                                           |
+| `CF_PIPELINE_NAME`           | The full path of the pipeline, including the project to which it is assigned, if any.                 |
+| `CF_BUILD_URL`               | The URL to the build in Codefresh.                                                                    |
+| `CF_STEP_NAME`               | The name of the step, for example, "MyUnitTests".                                                     |
+| `CF_BRANCH`                  | The branch name or tag of the Git repository associated with the main pipeline at the time of execution. |
+| `CF_REVISION`                | The revision of the Git repository of the main pipeline, at the time of execution.                   |
 
-[Full list of build environment variables provided by Codefresh][1]
 
+For a comprehensive list of environment variables set by Codefresh for every build, see the [official Codefresh documentation][101].
 
-[1]: https://codefresh.io/docs/docs/pipelines/variables/
-{{% /tab %}}
-{{% tab "GitHub Actions" %}}
 
-- `GITHUB_ACTION`
-- `GITHUB_SERVER_URL`
-- `GITHUB_RUN_ID`
-- `GITHUB_RUN_NUMBER`
-- `GITHUB_RUN_ATTEMPT`
-- `GITHUB_WORKFLOW`
-- `GITHUB_WORKSPACE`
-- `GITHUB_REPOSITORY`
-- `GITHUB_SHA`
-- `GITHUB_HEAD_REF`
-- `GITHUB_REF`
-- `GITHUB_JOB`
+[101]: https://codefresh.io/docs/docs/pipelines/variables/
 
-[Full list of build environment variables provided by GitHub Actions][1]
+{{% /tab %}}
+{{% tab "GitHub Actions" %}}
 
+| Environment Variable       | Description                                                                                           |
+|----------------------------|-------------------------------------------------------------------------------------------------------|
+| `CI`                       | Always set to `true`, indicating a CI environment.                                                   |
+| `GITHUB_ACTION`            | The name of the action currently running, or the id of a step. For example: `repo-owner_name-of-action-repo`. |
+| `GITHUB_ACTION_PATH`       | The path where an action is located (only supported in composite actions).                           |
+| `GITHUB_ACTION_REPOSITORY` | The owner and repository name of the action being executed. For example: `actions/checkout`.              |
+| `GITHUB_ACTIONS`           | Always set to `true` when GitHub Actions is running the workflow.                                    |
+| `GITHUB_ACTOR`             | The name of the person or app that initiated the workflow. For example: `octocat`.                       |
+| `GITHUB_ACTOR_ID`          | The account ID of the person or app that triggered the workflow. For example: `1234567`.                 |
+| `GITHUB_API_URL`           | The API URL. For example: `https://api.github.com`.                                                      |
```

Comment:
This list is a lot longer than the list that we originally had. Most of the new environment variables are never used by tracers, for example `GITHUB_API_URL`, or `RUNNER_` vars.

Maybe let's keep in this list only the original environment variables that are actually used by tracers?